### ISSUE TITLE
[sabredav] PDOException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#417]
```
Token: fe66ba63af40a34bb712f40152c6e8c1 (Don't modify this token!)
Version: 2.5 (31-10-2012)
PHP: 5.3.3-7 squeeze14
RCMAIL: 0.8.2
Database: mysqli
SERVER: Apache
----
I.  Issue Description:

II. Steps to reproduce the Issue:
1. Install using the official installation howto

After following the howto on the website this is the error that is screwing up 
everything... carddav works though...
```

Original issue reported on code.google.com by `erwin.wi...@gmail.com` on 4 Nov 2012 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
whoops forgot screenshot...
```

Original comment by `erwin.wi...@gmail.com` on 4 Nov 2012 at 9:05

Attachments:
- [Screenshot from 2012-11-04 21:56:35.png](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-417/comment-1/Screenshot from 2012-11-04 21:56:35.png)

**Comment by GoogleCodeExporter** [comment]
```
Our SabreDAV plugin is based on SabreDAV 1.6.x. You are using 1.7.1. Wait for 
an update; it is already scheduled.

Meanwhile you should install 1.6.x if you want to deploy our plugin.
```

Original comment by `roland.l...@gmail.com` on 4 Nov 2012 at 9:33
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in sabredav 3.0. Don't miss to read our update notes:

http://myroundcube.com/myroundcube-plugins/sabredav-plugin
```

Original comment by `roland.l...@gmail.com` on 15 Dec 2012 at 1:43
- Changed state: **Fixed**
